Multisig/UI/UI Library/String+Highlight.swift

```diff
@@ -37,25 +37,13 @@ extension String {
     func highlightRange(originalStyle: GNOTextStyle = .secondary,
                         highlightStyle: GNOTextStyle = .primary,
                         textToHighlight: String? = nil) -> NSAttributedString {
-            var style = NSMutableParagraphStyle()
-            style.lineSpacing = 10
-
-            var attributedString = NSMutableAttributedString(
-            string: self,
-            attributes: [
-                .font: UIFont.gnoFont(forTextStyle: originalStyle),
-                .foregroundColor: originalStyle.color!,
-                .kern: -0.41,
```

Comment:
I removed it because I thought that in Figma it is an 'automatic' style, i.e. we just use our system font without kerning.

Kerning is set in the GNOTextSTyle